docs/source/getting-started.rst

```diff
@@ -95,7 +95,15 @@ Deploy
   ./manage.py configure aws_account_id 111111111111  # Replace with your 12-digit AWS account ID
   ./manage.py configure prefix NAME                  # Choose a unique name prefix (alphanumeric characters only)
 
-2. Build the StreamAlert infrastructure for the first time:
+2. Configure terraform state locking:
```

Comment:
I think @ryandeivert  has a good point. Is the terraform state locking mandatory or optional?

Reply:
I suggested that we _not_ make it configurable, since I could not come up with an argument as to why anyone would explicitly not want it. do you have an opinion @chunyong-lin ?

Reply:
I left it as configure because there is still some configuration needed on there end, I.E adding the proper permissions to whatever role the users that need to deploy are in. This is definitely a mandatory feature.

Reply:
I'm of the opinion that there is a LOT of permissions a user needs to have in order to even deploy streamalert (all the actions/api calls that terraform makes under the hood require the user to have all applicable permissions...). I don't think we should be highlighting this here, when we make no mention of the super set of permissions needed for deployment in general

eventually, we should publish a least-privileged IAM role template that people can use, but we already mention the user should have DynamoDB permissions in general. if they don't have the right privileges, they'll get an `AccessDenied` exception and be able to address on their end. we shouldn't try to account for this yet

Reply:
@ryandeivert Ah yeah that makes sense. DynamoDB is already in that list. Since there is no action for the user to take, I'm going to make this a note in the getting started section and remove it from deployment steps.